WiFlyHQ.h

```diff
@@ -312,23 +312,23 @@ class WiFly : public Stream {
     boolean setFtpMode(uint8_t mode);
 
     boolean ftpGet(
-	const char *addr,
-	const char *dir,
-	const char *user,
-	const char *password,
-	const char *filename);
+    const char *addr,
```

Comment:
This change has removed the indent that offsets the parameters from the method definition.  Did you mean to do this?